twake/backend/node/src/services/workspaces/services/workspace/service.ts

```diff
@@ -63,6 +75,21 @@ export class WorkspaceService implements WorkspaceServiceAPI {
       WorkspacePendingUser,
     );
 
+    const workspaceCountersRepository = await this.database.getRepository<WorkspaceCounterEntity>(
+      WorkspaceCounterEntityType,
+      WorkspaceCounterEntity,
+    );
+
+    this.workspaceCounter = await this.counters.getCounter<WorkspaceCounterEntity>(
+      workspaceCountersRepository,
+    );
+
+    this.workspaceCounter.reviseCounter(async (pk: WorkspaceCounterPrimaryKey) => {
+      return this.workspaceUserRepository
+        .find({ workspace_id: pk.id })
+        .then(a => a.getEntities().length);
+    });
```

Comment:
This will not work if there is pagination (more than 100 members) could you add a loop over the pages like we did here ? https://github.com/stephanevieira75/Twake/blob/1bcd2ad9eebde1fb087be77a6b1398aed9adaba2/twake/backend/node/src/cli/cmds/migration_cmds/php-message/message-migrator-service.ts#L47

Reply:
👀
sure!